clockwork.go

```diff
@@ -41,20 +41,36 @@ func NewRealClock() Clock {
 	return &realClock{}
 }
 
+// FakeClockOption is an option for configuring a FakeClock.
+type FakeClockOption func(*fakeClock)
+
+// WithSynchronousAfterFunc is a FakeClockOption which ensures that the callback to AfterFunc is called synchronously
+// instead of in a separate goroutine. If true, this option will cause any Advance calls to block until the callback
+// from all fired timers has completed.
+func WithSynchronousAfterFunc(b bool) FakeClockOption {
```

Comment:
On a first-pass, I don't think the argument is needed. The name of the option communicates the intent: if present, the AfterFunc is sync, if absent, AfterFunc is async (the default).

Reply:
Removed, thanks